frontend/src/components/Administration/UserList/User.tsx

```diff
@@ -47,7 +48,7 @@ const User: React.SFC<{ user: IUser, isAdmin: boolean }> = props => {
                 {email}
             </TableCell>
             <TableCell>
-                {role}
+                <RoleView role={user.role}/>
```

Comment:
Místo `user.role` použít jen `role`, někde to nad tím muselo projít destrukcí.

Reply:
Fixed